mlibrary/search
# [LIBSEARCH-1042] Update Articles & Catalog medium view

# Overview
Catalog and Articles records will be receiving [new metadata](https://github.com/mlibrary/spectrum/pull/200) for their medium views. In preparation for this, `Metadata` has been altered to trim the necessary information. Not only that, but all components relating to showing metadata have been rewritten to be higher performing and use less code.

This pull request fixes [LIBSEARCH-1042](https://mlit.atlassian.net/browse/LIBSEARCH-1042).

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Poke around the site and see if the records show normally.
